wait/host_port.go

```diff
@@ -178,6 +187,15 @@ func (hp *HostPortStrategy) WaitUntilReady(ctx context.Context, target StrategyT
 		}
 	}
 
+	if hp.skipExternalCheck {
+		return nil
+	}
+
+	ipAddress, err := target.Host(ctx)
+	if err != nil {
+		return err
```

Comment:
suggestion: I know this was just moved but we're making sure all new code wraps returned errors to make it easier to identify the source.
```suggestion
		return fmt.Error("host: %w", err)
```

Reply:
Done